packages/affine/model/src/consts/line.ts

```diff
@@ -0,0 +1,27 @@
+import type { UnionArray } from '../utils/zod.js';
+
+import { createZodUnion } from '../utils/zod.js';
+
+export enum LineColor {
+  Black = '--affine-palette-line-black',
+  Blue = '--affine-palette-line-blue',
+  Green = '--affine-palette-line-green',
+  Grey = '--affine-palette-line-grey',
+  Magenta = '--affine-palette-line-magenta',
+  Orange = '--affine-palette-line-orange',
+  Purple = '--affine-palette-line-purple',
+  Red = '--affine-palette-line-red',
+  Teal = '--affine-palette-line-teal',
+  White = '--affine-palette-line-white',
+  Yellow = '--affine-palette-line-yellow',
+}
```

Comment:
Need to ensure order in color palette.

<img width="485" alt="Screenshot 2024-08-21 at 21 53 09" src="https://github.com/user-attachments/assets/2c977867-8731-4695-826d-7f059ca7dd79">

Reply:
Thanks for your comments! I have changed the code and ensured the order of color list.